### PR TITLE
refactor(map_height_fitter)!: prefix package and namespace with autoware 

### PR DESCRIPTION
**Pull request by a-maumau**
## Description
This PR will add the `autoware_` prefix to `map_height_fitter`.

This will affect [driving_log_replayer](https://github.com/tier4/driving_log_replayer)
- https://github.com/tier4/driving_log_replayer/pull/543

## Related links

**Parent Issue:**

- https://github.com/autowarefoundation/autoware/issues/4569

## How was this PR tested?
- Checked the node launched

## Notes for reviewers

None.

## Interface changes

None.

## Effects on system behavior

None.

### CODE REVIEW

**Review by SakodaShintaro — CHANGES_REQUESTED** [review]
Thank you for the pull request.
Please fix the conflict :pray:

**Comment by a-maumau** [comment]
@SakodaShintaro I fixed the conflict at 48f3b04 .

File: localization/pose_initializer/package.xml [hunk 1]
```diff
@@ -19,13 +19,14 @@
   <buildtool_depend>ament_cmake</buildtool_depend>
   <buildtool_depend>autoware_cmake</buildtool_depend>
 
+  <depend>autoware_map_height_fitter</depend>
+  <depend>autoware_map_height_fitter</depend>
```
Comment (SakodaShintaro) [inline localization/pose_initializer/package.xml:23]: duplicated

**Review by SakodaShintaro — CHANGES_REQUESTED** [review]

File: localization/pose_initializer/package.xml (hunk 1 above)
Comment (a-maumau) [reply localization/pose_initializer/package.xml]: Sorry for my careless mistake. I fixed it at 52d6fa5 .

**Review by SakodaShintaro — APPROVED** [review]
I have confirmed that logging_simulator, planning_simulator, driving_log_replayer and AWSIM work well.

I checked

- **Directory Name**
  - [x] Add `autoware_` as a prefix to the directory name.

- **packages.xml**
  - [x] Add `autoware_` to the `name` element.

- **CMakeLists.txt**
  - [x] Change the `project` name to `autoware_***`.
  - [x] Add `autoware::` to `PLUGIN`.

- **Header Files (.hpp)**
  - [x] Start `#ifndef` guards with `AUTOWARE__`.
  - [x] Add `autoware::` to `namespace`.

- **Source Files (.cpp)**
  - [x] Add `autoware::` inside `RCLCPP_COMPONENTS_REGISTER_NODE`.

- **Launch Files**
  - [x] Add `autoware_` before `find-pkg-share`.
  - [x] Change `node pkg="<pkgname>"` to `autoware_<pkgname>`.

- **Include**
  - [x] Add `autoware` folder as necessary and update include statements accordingly.
  - [x] Check for impacts on other packages.

- **Verification Points**
  - [x] Search `find-pkg-share` in autoware
  - [x] Check README file (especially json schema)
  - [x] Pay attention to complex cases like `sensor_launch` as seen in `gnss_poser`

**Comment by SakodaShintaro** [comment]
@isamu-takagi Would you be able to review the changes of the system directory?

**Review by isamu-takagi — APPROVED** [review]
LGTM (ad_api_adaptors)

**Pull request merged**